biome.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "vcs": {
+    "enabled": true,
+    "clientKind": "git",
+    "useIgnoreFile": true,
+    "defaultBranch": "main"
+  },
+  "files": {
+    "include": ["*.ts", "*.tsx", "*.json"],
```

Comment:
.svelte?

Reply:
Oh true I think biome has partial support for svelte, I remember Ewan saying something about it messing up some stuff though... definitely something to look into